__tests__/app.test.js

```diff
@@ -1,8 +1,18 @@
 const endpointsJson = require("../endpoints.json");
+
 /* Set up your test imports here */
 
+const app = require('../app');
+const request = require('supertest');
+const db = require('../db/connection');
+const seed = require('../db/seeds/seed');
+const { topicData, userData, articleData, commentData } = require("../db/data/test-data/index");
+
 /* Set up your beforeEach & afterAll functions here */
 
+beforeEach(() => seed({ topicData, userData, articleData, commentData }));
```

Comment:
Is there a reason we are destructuring all these methods only to put them back on an object and pass it through to the seed?